### PR TITLE
Census individual are these details correct

### PR DESCRIPTION
**Pull request by jonnyshaw89**
### What is the context of this PR?
Fixes https://github.com/ONSdigital/eq-survey-runner/issues/625

### CODE REVIEW

**Review by ajmaddaford — APPROVED** [review]

File: tests/functional/spec/census/issues/individual-Q1-are-these-details-correct.spec.js [hunk 1]
```diff
@@ -0,0 +1,106 @@
+import chai from 'chai'
+import {openAndStartCensusQuestionnaire} from '../../../helpers'
+
+import PermanentOrFamilyHome from '../../../pages/surveys/census/household/permanent-or-family-home.page.js'
```
Comment (iwootten) [inline tests/functional/spec/census/issues/individual-Q1-are-these-details-correct.spec.js:3]: It looks like many of these imports aren't actually used. I count 17 pages in the tests, with 71 imports. Can they be tidied up?

**Review by iwootten — CHANGES_REQUESTED** [review]

File: tests/functional/spec/census/issues/household-member-details-q1-are-these-details-correct.spec.js [hunk 2]
```diff
@@ -0,0 +1,106 @@
+import chai from 'chai'
+import {openAndStartCensusQuestionnaire} from '../../../helpers'
+
+import PermanentOrFamilyHome from '../../../pages/surveys/census/household/permanent-or-family-home.page.js'
```
Comment (iwootten) [inline tests/functional/spec/census/issues/household-member-details-q1-are-these-details-correct.spec.js]: It looks like many of these imports aren't actually used. I count 17 pages in the tests, with 71 imports. Can they be tidied up?

**Review by iwootten — CHANGES_REQUESTED** [review]

File: tests/functional/spec/census/issues/household-member-details-q1-are-these-details-correct.spec.js (hunk 2 above)
Comment (jonnyshaw89) [reply tests/functional/spec/census/issues/household-member-details-q1-are-these-details-correct.spec.js]: Yeh i guess so, they are auto generated with the page models. Have we excluded tests from any code checking? as those things normally go nuts over this.

Comment (jonnyshaw89) [reply tests/functional/spec/census/issues/household-member-details-q1-are-these-details-correct.spec.js]: Have removed unused

**Review by iwootten — APPROVED** [review]

**Pull request merged**